### ISSUE TITLE
[Feature Request] Import and export settings.

### ISSUE THREAD

**Issue by tnaseem** [mdjekic/pro4trello#24]
I just left a similar feature request on the Play Store and just realised you're on Github :)

I use a couple of machines and Wavebox on both, as well as Chrome. It would be great to export settings (especially the card colours) and import them to the others.

Not sure if this is possible, but it would be a great addition.

**Comment by mdjekic** [comment]
Hi! You can export settings from any board (and import them into another). Check your board settings and you will see it at the top.

**Comment by tnaseem** [comment]
Ha! How did I miss that!? Many thanks :)

**Comment by tnaseem** [comment]
Just an update...

If I copy the settings to the clipboard from Chrome and then immediately attempt to paste it back in, it gives me the error - "Oops... Bad JSON in Clipboard...'.

I'm wondering if it's anything to do with the "undefined" setting in the file (see above).

Hope all this helps!

**Comment by tnaseem** [comment]
Great update on 3.3.

I just tried the export and import clipboard and this time it's accepting the JSON file.

However, although the board settings change, the colours for the board columns are not being applied still.

Exported JSON:
```
{"beautify-markdown":false,"compact-cards":false,"custom-background":false,"custom-background-input":"","custom-css":false,"custom-css-input":"","filters":{},"full-screen-cards":false,"hide-activity-entries":false,"hide-add-list":false,"labels-size":"medium","listEnchancements":{"backlog":{"background":"#ffffff50"},"current-sprint":{"background":"#00000020"},"done":{"background":"#00ff0050"},"done0706-2020":{"background":"#00ff0050"},"done1406-2020":{"background":"#00ff0050"},"fixed":{"background":"#00ff0050"},"in-progress":{"background":"#00000030"},"internalissues-tweaks":{"background":"#ff000080"},"roadmap":{"background":"#ffffff50"}},"parse-hashtags":false,"parse-labels":false,"parse-markup":true,"parse-points":false,"parse-price-entries":false,"parse-priority-marks":false,"parse-projects":false,"parse-time-entries":false,"show-list-stats":false,"show-list-stats-progressbar":false,"visible-card-numbers":false,"visible-labels":false}
```

_parse-markup_ gets applied, but none of the colours.

**Comment by mdjekic** [comment]
Yeah, when it comes to colors and layout in general, it's kind of a personal experience. While sharing settings in general makes sense for teams, these significant visual changes are left for personalisation - to individuals. I think this was my way of thinking when I did not include them while processing imported settings. However, I will look into the code and figure out if it's ez and maybe just import all settings.

**Comment by tnaseem** [comment]
Cheers chap. Great work though. I've just donated to the cause :)
